src/main/java/com/foundation/model/Search.java

```diff
@@ -0,0 +1,58 @@
+/*
+ *  @(#)Search.java Copyright (c) 2018 Jalasoft.
+ *  2643 Av Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ *  All rights reserved.
+ *
+ *  This software is the confidential and proprietary information of
+ *  Jalasoft, ("Confidential Information").  You shall not
+ *  disclose such Confidential Information and shall use it only in
+ *  accordance with the terms of the license agreement you entered into
+ *  with Jalasoft.
+ *
+ */
+
+package com.foundation.model;
+
+import java.io.*;
```

Comment:
Import only the classes that are going to be used; try to avoid the * wildcard to import everything.